### ISSUE TITLE
'informed consent process' as a pipeline of subprocesses vs 'consent' as the last subprocess

### ISSUE THREAD

**Issue by yongqunh** [ICO-ontology/ICO#15]
The 'informed consent process' (http://purl.obolibrary.org/obo/OBI_0000810) is defined as 'A planned process in which a person or their legal representative is informed about key facts about potential risks and benefits of a process and makes a documented decision as to whether the person in question will participate.' 

Based on this definition, it appears that such an 'informed consent process' includes a collection of several subprocesses such as subject asking questions about risks and benefits, and making a decision (consent to participate or drop to participate). In this case, we need to represent individual subprocesses as well. for example, 
 
'informed consent process' = 'subject asking questions about risks' + 'PI (or other) answering question' + ... + 'subject consent (or refusal) to participating' 

If we use this model, the last step may be called 'informed consent' which includes different types (e.g., assent, verbal consent, short form consent, ...)

See references:
https://oprs.usc.edu/files/2017/04/Informed-Consent-Booklet-4.4.13.pdf 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3777303/ 

I proposed this design in our CTSOG informed consent workshop:
http://ncorwiki.buffalo.edu/index.php/Ontology_of_Informed_Consent:_An_Approach_to_Specimen_and_Data_Sharing

and generated the new term 'informed consent' (ICO_0000155) and many subclass terms under it:
http://purl.obolibrary.org/obo/ICO_0000155

However, we have not got a general consensus on this design. Therefore, we will need more discussion on this issue. 
Thanks.  - Oliver

**Comment by linikujp** [comment]
What is the difference between "informed consent" and "informed consent
process"?
As "informed consent" is a subclass of process, using this label is very
confusing people. I suggest to use "consenting process" instead.

Thanks,
Asiyah

**Comment by yongqunh** [comment]
Yes, 'consenting process' or just 'consent process' may be better. So we can have:
- 'informed consent process': which is a collection of processes
- 'consent process': which is the last step of the 'informed consent process'.

then, does the 'consent process' mean that the subject agrees/consents? or does it mean that the subject can agree/consent or disagree/disconsent?

**Comment by linikujp** [comment]
Hi Oliver,

You made a good point. The consent has to be voluntary. We may need to add
"Voluntary consenting process" and a decline process of consent as well.

Thanks,
Asiyah
